idem_azurerm/exec/azurerm/authorization/permission.py

```diff
@@ -6,7 +6,7 @@
 
 :maintainer: <devops@eitr.tech>
 :configuration: This module requires Azure Resource Manager credentials to be passed as keyword arguments
-to every function in order to work properly.
+    to every function in order to work properly.
```

Comment:
Should this be "to every function or via acct in order to work properly." like the other files?

Reply:
yup. u r smrt